src/ParticleEmitter.cc

```diff
@@ -96,6 +96,13 @@ class sdf::ParticleEmitterPrivate
   /// \brief Topic used to update particle emitter properties at runtime.
   public: std::string topic = "";
 
+  /// \brief Particle scatter ratio. This is used to determine the ratio of
+  /// particles that will be detected by sensors. Increasing the ratio
+  /// means there is a higher chance of particles reflecting and interfering
+  /// with depth sensing, making the emitter appear more dense. Decreasing the
+  /// ratio decreases making it appear less dense. This value should be > 0.
```

Comment:
My previous comment in `sdf/1.7/particle_emitter.sdf` also applies here (fixing the documentation and `> 0` vs `>= 0`)

Reply:
b89e8b3